e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/DemographicsTest.groovy

```diff
@@ -30,4 +34,25 @@ class DemographicsTest extends Specification {
         then:
         responseBody == "Server Error"
     }
+
+    def "payload file check"() {
+
+        when:
+        def responseBody = Client.post("/v1/etor/demographics", newbornPatientJsonFileString)
+        def sentPayload = SentPayloadReader.read()
+        def parsedResponseBody = JsonParsing.parse(responseBody, Map.class)
+
+        def parsedSentPayload = JsonParsing.parse(sentPayload, Map.class)
+
+        then:
+
+        parsedSentPayload.entry[0].resource.resourceType == "MessageHeader"
+        parsedSentPayload.entry[2].resource.resourceType == "ServiceRequest"
+
+        parsedSentPayload.entry[1].resource.resourceType == "Patient"
+        parsedSentPayload.entry[1].resource.id == "infant-twin-1"
+
+        parsedSentPayload.entry[1].resource.identifier[0].value == parsedResponseBody.patientId
+        parsedSentPayload.entry[1].resource.resourceType + "/" + parsedSentPayload.entry[1].resource.id == parsedResponseBody.fhirResourceId
```

Comment:
I did this concatenation to show that the combined data would match the "fhirResourceId"